src/helpers/__tests__/flare.test.ts

```diff
@@ -4,13 +4,20 @@ import process from 'process'
 import axios from 'axios'
 import FormData from 'form-data'
 
+import {PROJECT_FILES} from '../../commands/lambda/constants'
 import {CI_SITE_ENV_VAR, SITE_ENV_VAR} from '../../constants'
 
-import {getEndpointUrl, sendToDatadog} from '../flare'
+import {getEndpointUrl, getProjectFiles, sendToDatadog, validateFilePath} from '../flare'
+import * as flareModule from '../flare'
 
-jest.mock('jszip')
+import {MOCK_CWD} from './fixtures'
+
+// Mocks
 jest.mock('fs')
+process.cwd = jest.fn().mockReturnValue(MOCK_CWD)
```

Comment:
Wasn't it possible to do `jest.spyOn(process, 'cwd').mockReturnValue()`?

Reply:
Yes, that is possible. I have changed all appearances of `process.cwd = ...`. Thanks for the comment!